exodus_lambda/functions/origin_request.py

```diff
@@ -101,6 +101,14 @@ def handler(self, event, context):
                 request["uri"] = (
                     "/" + query_result["Items"][0]["object_key"]["S"]
                 )
+                if query_result["Items"][0]["content_type"]["S"]:
+                    content_type = query_result["Items"][0]["content_type"][
```

Comment:
Guess this could be slightly cleaner if content_type is before the "if", that way you don't have to write the entire query_result[...] expression twice.